server/controllers/finance/feeCenter.js

```diff
@@ -0,0 +1,263 @@
+/**
+* The /fee_centers HTTP API endpoint
+*
+* @module finance/feeCenters
+*
+* @description This controller is responsible for implementing all crud and others custom request
+* on the fee_center table through the `/fee_centers` endpoint.
+*
+* @requires lib/db
+**/
+
+
+var db = require('../../lib/db');
+var NotFound = require('../../lib/errors/NotFound');
+
+/**
+* Returns an array of fee centers
+*
+* @param {object} req The express request object
+* @param {object} res The express response object
+* @param {object} next The express object to pass the control to the next middleware
+*
+* @example
+* // GET /fee_centers : Get list of fee_centers
+* var feeCenters = require('finance/fee_centers');
+* feeCenters.list(req, res, next);
+*/
+
+function list (req, res, next) {
+  'use strict';
+
+  var sql =
+    'SELECT fc.id, fc.label, fc.is_cost FROM fee_center AS fc ';
+  var filter = '', ordering = 'ORDER BY fc.label;';
+
+  if (req.query.full === '1') {
```

Comment:
I think `req.query.detailed` makes a lot more sense.  It will also be standardized once @sfount's PR lands in the filter library.  Could you use `req.query.detailed` instead?